deeplake/core/dataset/dataset.py

```diff
@@ -1462,6 +1463,7 @@ def read_only(self, value: bool):
     @deeplake_reporter.record_call
     def pytorch(
```

Comment:
Remove *args and **kwargs from public API. MMDet intergration should use `to_pytorch` directly.